libbeat/docs/version.asciidoc

```diff
@@ -1,7 +1,8 @@
-:stack-version: 7.0.0-alpha1
-:doc-branch: master
-:go-version: 1.9.2
-:release-state: unreleased
-:python: 2.7.9
-:docker: 1.12
-:docker-compose: 1.11
+:stack-version:   7.0.0-alpha1
```

Comment:
As we use these variables in other places (yes, we shouldn't) I wonder if this could cause some issues with spaces in front.

@exekias These are used in the k8s configs too.

Reply:
Yes it seems there is an issue with the extra spaces https://travis-ci.org/elastic/beats/jobs/325272871#L717, @lcawl could you remove them?